client/app/components/string/WrapOverflow.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Return first X characters of any string and add
+ * tail if needed.
+ */
+angular.module("filters").filter("wrapOverflow", () => function (value, max, tail) {
```

Comment:
You should use `_.trunc`

Reply:
I guess, it would have the same exact effect but it would add a dependency with lodash, which would not be the end of the world

Reply:
Please @cbourgois could you update your review so I can merge :-)
Thanks